nautobot/docs/additional-features/jobs.md

```diff
@@ -112,16 +115,19 @@ Default: `False`
 
 A boolean that will mark this job as requiring approval from another user to be run. For more details on approvals, [please refer to the section on scheduling and approvals](./job-scheduling-and-approvals.md).
 
-#### `commit_default`
+#### `dryrun_default`
 
-Default: `True`
++/- 2.0.0
+    The `commit_default` field was renamed to `dryrun_default` and the default value was changed from `True` to `False`.
```

Comment:
Need to also document that, like `read_only`, it no longer intrinsically affects the behavior of the job (except in allowing runs without approval), i.e. it no longer controls anything to do with rollback or prevention of DB changes.